rosidl_typesupport_connext_cpp/resource/msg__type_support.cpp.em

```diff
@@ -149,35 +149,52 @@ convert_ros_message_to_dds(
 bool
 publish__@(spec.base_type.type)(
   void * untyped_topic_writer,
-  const void * untyped_ros_message)
+  ConnextStaticMessageHandle * message_handle)
```

Comment:
Why should this not be const anymore? Why should the `untyped_dds_message` be stored in this handle? Couldn't it be just a local variable?